assets/layers/ticket_machine/ticket_machine.json

```diff
@@ -0,0 +1,235 @@
+    "en": "Ticket Machines"
+  },
+  "description": {
+    "en": "Find ticket machines for public transport tickets"
+  },
+  "source": {
+    "osmTags": {
+      "and": [
+        "amenity=vending_machine",
+        "vending=public_transport_tickets"
+      ]
+    }
+  },
+  "minzoom": 19,
+  "title": {
+    "render": {
+      "en": "Ticket Machine"
+    }
+  },
+  "tagRenderings": [
+    "images",
+    "level",
+    {
+      "id": "operator",
+      "question": {
+        "en": "Who is the operator of this ticket machine?"
+      },
+      "render": {
+        "en": "This ticket machine is operated by {operator}"
+      },
+      "freeform": {
+        "key": "operator",
+        "type": "string",
+        "placeholder": {
+          "en": "Name of the operator"
+        }
+      },
+      "mappings": [
+        {
+          "if": "operator=Nederlandse Spoorwegen",
+          "then": {
+            "en": "Dutch Railways (NS)",
+            "nl": "Nederlandse Spoorwegen (NS)"
+          },
+          "addExtraTags": [
+            "operator:wikidata=Q23076"
+          ],
+          "hideInAnswer": "_country!=nl"
+        }
+      ]
+    },
+    {
+      "id": "payment-options",
+      "question": {
+        "en": "How can you pay for tickets at this ticket machine?"
+      },
+      "multiAnswer": true,
+      "mappings": [
+        {
+          "if": "payment:coins=yes",
+          "ifnot": "payment:coins=no",
+          "icon": "./assets/layers/ticket_machine/coins.svg",
+          "then": {
+            "en": "You can pay using coins"
+          }
+        },
+        {
+          "if": "payment:notes=yes",
+          "ifnot": "payment:notes=no",
+          "icon": "./assets/layers/ticket_machine/notes.svg",
+          "then": {
+            "en": "You can pay using bank notes"
+          }
+        },
+        {
+          "if": "payment:cash=yes",
+          "ifnot": "payment:cash=",
+          "icon": "./assets/tagRenderings/cash.svg",
+          "then": {
+            "en": "You can pay here using coins and bank notes"
+          },
+          "hideInAnswer": true
+        },
+        {
+          "if": "payment:debit_cards=yes",
+          "ifnot": "payment_debit_cards=no",
+          "icon": "./assets/tagRenderings/payment_card.svg",
+          "then": {
+            "en": "You can pay here using debit cards"
+          }
+        },
+        {
+          "if": "payment:credit_cards=yes",
+          "ifnot": "payment:credit_cards=no",
+          "icon": "./assets/tagRenderings/payment_card.svg",
+          "then": {
+            "en": "You can pay here using credit cards"
+          }
+        },
+        {
+          "if": "payment:cards=yes",
+          "ifnot": "payment:cards=",
+          "icon": "./assets/tagRenderings/payment_card.svg",
+          "then": {
+            "en": "You can pay here using cards"
+          },
+          "hideInAnswer": true
+        }
+      ]
+    },
+    {
+      "id": "denominations-coins",
```

Comment:
This one would be well-suited for the shared questions.json as well

Reply:
Moved this one over as well